pages/linux/eyewitness.md

```diff
@@ -0,0 +1,24 @@
+# EyeWitness
+
+> Python based tool to gather screenshots of websites, as well as additional information such as header info and default credentials if possible.
+> More information: <https://www.christophertruncer.com/eyewitness-triage-tool>.
+
+- Scan a single website using `--single` flag:
```

Comment:
```suggestion
- Scan a specific website:
```

- Optional fix: `a specific` may sound better.
- Optional fix: there is no need to explicitly mention flag names until it's unclear (definition is missing in our guides).